tests/integration/test_runner_manager_openstack.py

```diff
@@ -1,5 +1,5 @@
-# Copyright 2024 Canonical Ltd.
-# See LICENSE file for licensing details.
+#  Copyright 2024 Canonical Ltd.
+#  See LICENSE file for licensing details.
```

Comment:
Just out of curiosity, is the extra space intended?

Reply:
(as in this is what the linter/auto formatters prefer)

Reply:
I moved the file to the github-runner-manager and checked it again here. I am not sure what is causing the extra space.